### ISSUE TITLE
Issue with Node Synchronization and Mining in Blockdag

### ISSUE THREAD

**Issue by Cupacapi** [BlockdagNetworkLabs/blockdag-scripts#16]
Hello Blockdag Support Team,

I hope this message finds you well. I am reaching out because I am experiencing an issue with the synchronization of my node and mining on the Blockdag testnet. I have followed the official guide for configuration, but I continue to encounter the following warning in the node logs:
Side chain depth too large
Here are the steps I have followed so far:

    I cloned the blockdag-scripts repository and ran the installation script.

    I configured the config.toml file as follows:
[Node]
Network = "testnet"
DataDir = "/home/cupa/blockdag-scripts/linux/data"

[RPC]
Enable = true
User = "cupa"
Password = "4293"
I edited the docker-compose.yml file to include my mining address:
services:
  bdag:
    image: blockdagnetwork/bdag:latest
    container_name: blockdag-testnet-network
    ports:
      - "38131:38131"
      - "8130:8130"
      - "18545:18545"
      - "18546:18546"
      - "18150:18150"
    volumes:
      - ./bin/bdag/data:/bdag/data
      - ./bin/bdag/logs:/bdag/logs
    command: >
      ./bdag
      --testnet
      --rpclisten=0.0.0.0:38131
      --notls
      --rpcuser=test
      --rpcpass=test
      --miner
      --miningaddr=Tk6uE464y4tdarMnEfqsbhq1W5CCWQxKGGz5XNYkYpegqv3GyLsJu
      --generate
      --walletpass=test
      --evmenv="--http --http.port=18545 --http.addr=0.0.0.0 --http.api=net,web3,eth,bdag --ws --ws.origins=* --ws.port=18546"
      --miningstatesync
      --addpeer=/ip4/3.14.177.55/tcp/18150/p2p/16Uiu2HAmKr4kFun2dLwVZwjS3Q8VfkAcqFm4W94RyBYA3dQ4sn3n
I started the Docker containers and checked the logs. Here are the warnings that appear:
2024-11-12|17:37:58.044 [WARN ] Side chain depth too large          module=miner
Despite following these steps, the node continues to display this warning and does not seem to synchronize correctly. I would appreciate any guidance or solutions you can offer to resolve this issue.

Thank you for your time and assistance.

Best regards, [Cupa]

**Comment by IcePool23** [comment]
is it a vm you are running node on?
I am having same issue on vm.

**Comment by Cupacapi** [comment]
No, yo estoy corriendo el nodo en un disco físico de arranque dual Windows y Ubuntu lo que si tengo es que estoy vía wifi

**Comment by AlliTobba** [comment]
Everyone is having this issue, there was an update last friday which caused this.

**Comment by Cupacapi** [comment]
> Everyone is having this issue, there was an update last friday which caused this.

Hi Alli Tobba,

I recently posted an issue on GitHub regarding Blockdag mining, but responses there can take a while. Do you know of any forums or online communities where Blockdag miners share their experiences and tips?

Thanks!

**Comment by AlliTobba** [comment]
I wish there was one as well :-)
I sent a bunch of questions to support and testnet the only reply I got was, please visit wiki.blockdag.network
I really want to update the node to MainNet

**Comment by AlliTobba** [comment]
@urkishan when can we update to MainNet 💯

**Comment by Cupacapi** [comment]
> I wish there was one as well :-) I sent a bunch of questions to support and testnet the only reply I got was, please visit wiki.blockdag.network I really want to update the node to MainNet

Hi Alli,

Thanks for your response! It's good to know I'm not the only one looking for more resources.

I believe MainNet hasn't been launched yet. Do you have any updates on the timeline for that?

Best regards, Cupa

**Comment by AlliTobba** [comment]
@Cupacapi development is supposed to ready tomorrow so why not launch :-)
Don´t miss the AMA tomorrow

**Comment by Cupacapi** [comment]
Hi Alli, Thanks for getting back to me! I'm definitely looking forward to the AMA and will be keeping an eye out for it.

Would it be possible for us to connect through another medium for a more direct conversation? If not, we can discuss it tomorrow.

**Comment by AlliTobba** [comment]
@Cupacapi yes, I all for it.  
https://meet.google.com/hme-upft-ubc
@IcePool23, join us

**Comment by Cupacapi** [comment]
@AlliTobba  I didn't join the meeting yesterday because my spoken English isn't very fluent, and I sometimes have to use a translator. I'm Spanish, but I'll give it a try in the next meeting you hold.

**Comment by IcePool23** [comment]
> @Cupacapi yes, I all for it. https://meet.google.com/hme-upft-ubc @IcePool23, join us

ah didnt notice message.

**Comment by AlliTobba** [comment]
@Cupacapi find me on Instagram if you have an account.
AMA was disappointing, no major update or release.  And very disappointing that they will not support SHA256
And as we suspected, there is a bug regarding the nodes.  
https://blockdag.network/dev-releases/dev-release-162

Also, no appreciation to us for running the nodes on testnet 👎 :-(

**Comment by Cupacapi** [comment]
@AlliTobba I haven’t had the chance to watch the AMA yet; I couldn’t see it yesterday and they haven’t published the recording yet. The MainNet launch was scheduled for yesterday, but it has been postponed as I saw on the roadmap. My Instagram account is @cupacapital; I looked for you but couldn’t find you. If I didn’t read it wrong in the release, it talks about rewards, or did they mention it in the AMA?

**Comment by Cupacapi** [comment]
@AlliTobba I haven’t had the chance to watch the AMA yet; I couldn’t see it yesterday and they haven’t published the recording yet. The MainNet launch was scheduled for yesterday, but it has been postponed as I saw on the roadmap. My Instagram account is @cupacapital; I looked for you but couldn’t find you. If I didn’t read it wrong in the release, it talks about rewards, or did they mention it in the AMA?

**Comment by urkishan** [comment]
Closing this issue because it was related to `Alpha Testnet`.
